src/applications/personalization/profile-2/tests/e2e/profile.cypress.spec.js

```diff
@@ -0,0 +1,105 @@
+import { PROFILE_PATHS } from '../../constants';
+
+import mockUser from '../fixtures/users/user-36.json';
+import mockPaymentInfo from '../fixtures/payment-information/direct-deposit-is-set-up.json';
+import mockFeatureToggles from '../fixtures/feature-toggles.json';
+
+/**
+ *
+ * @param {boolean} mobile - test on a mobile viewport or not
+ *
+ * This helper:
+ * - loads the Profile, confirms the user is redirected to the correct URL, and
+ *   then performs an aXe scan
+ * - clicks through each item in the sub-nav, checks that the URL is correct,
+ *   and performs an aXe scan
+ */
+function checkAllPages(mobile = false) {
+  cy.visit('/profile');
+  if (mobile) {
+    cy.viewport('iphone-6');
+  }
+
+  // should show a loading indicator
+  cy.findByRole('progressbar').should('exist');
+  cy.findByText(/loading your information/i).should('exist');
+
+  // and then the loading indicator should be removed
+  cy.findByRole('progressbar').should('not.exist');
+  cy.findByText(/loading your information/i).should('not.exist');
+
+  // visiting /profile should redirect to profile/personal-information
+  cy.url().should(
+    'eq',
+    `${Cypress.config().baseUrl}${PROFILE_PATHS.PERSONAL_INFORMATION}`,
+  );
+
+  // make the a11y check on the Personal Info section
+  cy.injectAxe();
+  cy.checkA11y();
```

Comment:
Does our `cy.axeCheck` custom command work here?

Reply:
Thanks for catching this! Yes, it likely does. It just didn't work in my first attempt because I needed to call `injectAxe()` first. I will switch to our custom command.

Reply:
I think you'll still need to `injectAxe()` first. `axeCheck` is really just a wrapper over `checkA11y` with some consistent defaults.

Reply:
yep yep... So I blindly tried `axeCheck()` first, not knowing what I was doing. When I got weird errors I just tried to run the `cypress-axe` scanner without our wrapper based on their instructions. Once that passed I forgot to use our own wrapper :)